articles/azure-monitor/app/asp-net-dependencies.md

```diff
@@ -100,7 +100,7 @@ For ASP.NET applications, full SQL query is collected with the help of byte code
 | --- | --- |
 | Azure Web App |In your web app control panel, [open the Application Insights blade](../../azure-monitor/app/azure-web-apps.md) and enable SQL Commands under .NET |
 | IIS Server (Azure VM, on-prem, and so on.) | [Install Status Monitor on your server where application is running](../../azure-monitor/app/monitor-performance-live-website-now.md) and restart IIS.
-| Azure Cloud Service |[Use startup task](../../cloud-services/cloud-services-startup-tasks.md) to [Install Status Monitor](monitor-performance-live-website-now.md#download) |
+| Azure Cloud Service | Add [startup task to install StatusMonitor](../../azure-monitor/app/cloudservices.md#set-up-status-monitor-to-collect-full-sql-queries-optional) <br> **Note**: your app should be onboaded to ApplicationInsights SDK at build time by installing nuget packages for [ASP.NET](https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net) or [ASP.NET Core applicaitons](https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core.) |
```

Comment:
Some small suggested changes. I removed `**Note**` because the contributor guide for Docs.microsoft.com tends to treat the word `Note` as a reserved keyword so while this is not universally enforced, often we will get asked to pull uses of `Note` that aren't using the syntax that creates a dedicated `Note` box:

```
> [!NOTE]
> “By the way” info not critical to a task
```

Since you are adding the reference inline in a table it makes sense to not use the special `Note` syntax as it would not work, but to avoid the PR getting blocked on final merge I would either cut `Note` or replace with a word other than `Note`, `Important`, `Tip` or `Warning` which are often treated as reserved.

```suggestion
| Azure Cloud Service | Add [startup task to install StatusMonitor](../../azure-monitor/app/cloudservices.md#set-up-status-monitor-to-collect-full-sql-queries-optional) <br> Your app should be onboarded to ApplicationInsights SDK at build time by installing NuGet packages for [ASP.NET](https://docs.microsoft.com/azure/azure-monitor/app/asp-net) or [ASP.NET Core applications](https://docs.microsoft.com/azure/azure-monitor/app/asp-net-core.) |
```